pkg/grpc/universalapi/api_workflow.go

```diff
@@ -189,3 +189,69 @@ func (a *UniversalAPI) RaiseEventWorkflowAlpha1(ctx context.Context, in *runtime
 	}
 	return &runtimev1pb.RaiseEventWorkflowResponse{}, nil
 }
+
+func (a *UniversalAPI) PauseWorkflowAlpha1(ctx context.Context, in *runtimev1pb.WorkflowActivityRequest) (*runtimev1pb.WorkflowActivityResponse, error) {
```

Comment:
This, `TerminateWorkflowAlpha1`, and `ResumeWorkflowAlpha1` are pretty much identical, even the signature is the same. The only difference seems to be that one calls `Pause`, one calls `Terminate`, and one calls `Resume`.

Please consider merging them into a single shared function. Something like a private function that accepts `(ctx, in, method)` where `method` is like `workflowComponent.Pause`.

Less code = easier to maintain :)